stackslib/src/config/mod.rs

```diff
@@ -1500,25 +1499,25 @@ pub struct BurnchainConfig {
     ///   - However, the height specified in `epochs` for Epoch 2.1 takes precedence.
     /// ---
     /// @default: `None`
-    /// @notes:
+    /// @notes: |
     ///   - Applied only if [`BurnchainConfig::mode`] is not "mainnet".
     ///   - This is intended strictly for testing purposes.
     pub pox_2_activation: Option<u32>,
     /// Overrides the length (in bitcoin blocks) of the PoX reward cycle.
     /// ---
     /// @default: `None` (uses the standard reward cycle length for the mode)
-    /// @notes:
+    /// @units: bitcoin blocks
+    /// @notes: |
     ///   - Applied only if [`BurnchainConfig::mode`] is not "mainnet".
     ///   - This is intended strictly for testing purposes.
-    ///   - Units: bitcoin blocks.
     pub pox_reward_length: Option<u32>,
     /// Overrides the length (in bitcoin blocks) of the PoX prepare phase.
     /// ---
     /// @default: `None` (uses the standard prepare phase length for the mode)
-    /// @notes:
+    /// @units: bitcoin blocks
```

Comment:
i like the inclusion of this key, but i wonder if we should keep it more generic and add a descirption where appropiate?
for example - `bitcoin blocks` is really just an int. 
what do you think of keeping units generic like `string`, `int`, `list|vector` etc and possibly adding a desciption of what the int is?
ex:
```
/// @units: int (defined in bitcoin blocks)
```

the downside is it would add more burden to keep it updated and adding new structs would require a more deliberate approach. i can definitely see some drift if we're not careful - a PR may just say `int` or `bitcoin blocks` for the unit and it would be up to the reviewers to catch it. 

I think either way is fine, just curious what you think of this idea.

Reply:
valid point, and it can always be changed later. resolving